test/hotspot/jtreg/serviceability/monitoring/ThreadInfo/getLockOwnerName/getLockOwnerName.java

```diff
@@ -0,0 +1,415 @@
+    public static final int TS_CONTENDER_RUNNING = 3;  // contender is running
+    public static final int TS_RELEASER_RUNNING  = 4;  // releaser is running
+    public static final int TS_CONTENDER_BLOCKED = 5;  // contender is blocked
+    public static final int TS_READY_TO_RELEASE  = 6;  // ready to release the blocker
+    public static final int TS_DONE_BLOCKING     = 7;  // done blocking threadLock
+    public static final int TS_CONTENDER_DONE    = 8;  // contender has run; done
+
+    public static Object barrierLaunch = new Object();   // controls thread launch
+    public static Object barrierBlocker = new Object();  // controls blocker
+    public static Object barrierReleaser = new Object();  // controls releaser
+    public static Object threadLock = new Object();      // testing object
+
+    public static long count = 0;
+    public static boolean printDebug = false;
+    public volatile static int testState;
+
+    private static void log(String msg) { System.out.println(msg); }
+
+    native static int wait4ContendedEnter(getLockOwnerNameWorker thr);
+
+    public static void main(String[] args) throws Exception {
+        try {
+            System.loadLibrary(AGENT_LIB);
+            log("Loaded library: " + AGENT_LIB);
+        } catch (UnsatisfiedLinkError ule) {
+            log("Failed to load library: " + AGENT_LIB);
+            log("java.library.path: " + System.getProperty("java.library.path"));
+            throw ule;
+        }
+
+        int timeMax = 0;
+        if (args.length == 0) {
+            timeMax = DEF_TIME_MAX;
+        } else {
+            int argIndex = 0;
+            int argsLeft = args.length;
+            if (args[0].equals("-p")) {
+                printDebug = true;
+                argIndex = 1;
+                argsLeft--;
+            }
+            if (argsLeft == 0) {
+                timeMax = DEF_TIME_MAX;
+            } else if (argsLeft == 1) {
+                try {
+                    timeMax = Integer.parseUnsignedInt(args[argIndex]);
+                } catch (NumberFormatException nfe) {
+                    System.err.println("'" + args[argIndex] +
+                                       "': invalid timeMax value.");
+                    usage();
+                }
+            } else {
+                usage();
+            }
+        }
+
+        int retCode = run(timeMax, System.out);
+        if (retCode != 0) {
+            throw new RuntimeException("Test failed with retCode=" + retCode);
+        }
+    }
+
+    public static void logDebug(String mesg) {
+        if (printDebug) {
+            System.err.println(Thread.currentThread().getName() + ": " + mesg);
+        }
+    }
+
+    public static void usage() {
+        System.err.println("Usage: " + AGENT_LIB + " [-p][time_max]");
+        System.err.println("where:");
+        System.err.println("    -p       ::= print debug info");
+        System.err.println("    time_max ::= max looping time in seconds");
```

Comment:
`::=` doesn't seem to be a convention we use in help output other than in the other recent tests you've added.

Reply:
The convention for java tools seems to be to just use tabs to align the start of the argument descriptions:

```
   -p          print debug info
   time_max    max looping time in seconds
```

Reply:
Fixed.